src/main/java/com/j256/ormlite/db/PostgresDatabaseType.java

```diff
@@ -127,4 +131,18 @@ public boolean isCreateIfNotExistsSupported() {
 			return super.isCreateIfNotExistsSupported();
 		}
 	}
+
+	@Override
+	public void appendOffsetTimeType(StringBuilder sb, FieldType fieldType, int fieldWidth) {
+		sb.append("TIMESTAMP WITH TIME ZONE");
+	}
+
+	@Override
+	public FieldConverter getFieldConverter(DataPersister dataPersister, FieldType fieldType) {
+		// H2 doesn't support TIME WITH TIME ZONE
```

Comment:
Minor: this should refer to Postgres - not H2.

Reply:
Fixed, thanks